src/app/classes/stix/stix-object.ts

```diff
@@ -586,8 +533,8 @@ export abstract class StixObject extends Serializable {
           refs_fields.push('aliases');
         if (this.attackType == 'asset') refs_fields.push('relatedAssets');
         if (this.attackType == 'technique') refs_fields.push('detection');
-        if (this.attackType == 'campaign')
-          refs_fields.push('first_seen_citation', 'last_seen_citation');
+        // if (this.attackType == 'campaign')
+        //   refs_fields.push('first_seen_citation', 'last_seen_citation');
```

Comment:
Are these fields being validated elsewhere? If so, these lines should be removed